config/env/default.js

```diff
@@ -42,6 +42,11 @@ module.exports = {
   },
   logo: 'modules/core/client/img/brand/logo.png',
   favicon: 'modules/core/client/img/brand/favicon.ico',
+  illegalUsernames: ['meanjs', 'fwd', 'fwd:', 'reply', 'administrator', 'password', 'community',
+                   'unknown', 'anonymous', 'null', 'undefined', 'home', 'signup', 'signin', 'login',
+                   'edit', 'settings', 'demo', 'support', 'networks', 'profile', 'avatar', 'mini',
+                   'photo', 'account', 'api', 'modify', 'feedback', 'security', 'accounts', 'tribe', 'tag'
```

Comment:
@hyperreality `^[^.](?=.*[0-9a-z])[0-9a-z.\-_](?!.*(\.)\1).*[^.]{3,34}` can you please validate this? I am not really good with regex